code-security/admin_guide/scan-monitor/software-composition-analysis/software-composition-analysis.adoc

```diff
@@ -146,7 +146,7 @@ image::sca-10.png[width=600]
 
 ==== Supply Chain
 
-As a remediation for sub-dependent packages, you can view and analyze the dependency tree on *Code Security > Supply Chain*. If the packages have direct dependencies irrespective of their placement in the dependency tree, Prisma Cloud offers solutions to these vulnerabilities. Here you can also choose to remediate the vulnerability by submitting a single PR (Pull Request) for all packages with vulnerabilities on the graph.
+As a remediation for sub-dependent packages, you can view and analyze the dependency tree on *Code Security > xref:../supply-chain-security.adoc[Supply Chain]*. If the packages have direct dependencies irrespective of their placement in the dependency tree, Prisma Cloud offers solutions to these vulnerabilities. Here you can also choose to remediate the vulnerability by submitting a single PR (Pull Request) for all packages with vulnerabilities on the graph.
```

Comment:
@anaghapamidi did adding the xref in the UI path work?

Please confirm that you were able to run panconv and create a PDF without errors

Reply:
@Pubs-MV yes the xref in the UI path works and the PDF is being generated smoothly.